RapidReact/src/main/java/frc/robot/subsystem/ClimberSubsystem.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystem;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import frc.robot.config.Config;
+import frc.robot.log.LogLevel;
+
+public class ClimberSubsystem extends BitBucketsSubsystem {
+
+  private WPI_TalonSRX climber;
+  private boolean enabledClimber;
+  private boolean disabledClimber;
+  private boolean fixedHook;
+  private boolean elevatorToggle;
+  private boolean elevatorExtend;
+  private boolean elevatorRetract;
+  private boolean autoClimb;
+  private boolean toggleState;
+  DoubleSolenoid climberSolenoid;
+
+  public ClimberSubsystem(Config config) {
+    super(config);
+  }
+
+  @Override
+  public void init() {
+    climber = new WPI_TalonSRX(Config.CLIMBER_MOTOR_ID);
+  }
+
+  @Override
+  public void periodic() {}
+
+  @Override
+  public void disable() {}
+
+  public void enableClimber() { // uses 2 PS button
+    enabledClimber = true;
+    logger().logString(LogLevel.GENERAL, "climb_state", "climberEnabled");
+  }
+
+  public void disableClimber() {
+    disabledClimber = true;
+    logger().logString(LogLevel.GENERAL, "climb_state", "climberDisabled");
+  }
+
+  public void fixedHookToggler() { //uses R1 button
+    fixedHook = true;
```

Comment:
Make this like toggleState.

We should rename toggleState to climbToggleState and use fixedHookToggleState to toggle the fixed hook with a separate solenoid.